lib/backend/src/Obelisk/Backend.hs

```diff
@@ -58,19 +70,37 @@ data GhcjsApp route = GhcjsApp
   , _ghcjsApp_value :: !(Frontend route)
   }
 
+type GhcjsWidgets r = GhcjsWidgetsImpl (Text -> FrontendWidgetT r ())
+
+-- | Widgets used to load all.js on the frontend
+data GhcjsWidgetsImpl a = GhcjsWidgets
```

Comment:
Hm, I'm inclined to suggest that we drop the type alias and just make the ctor consistent name.